### PR TITLE
Client UI skeleton

### PR DESCRIPTION
**Pull request by talivneh**

### CODE REVIEW

File: src/client/src/routes/QuizePage.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Page from "../components/Page/Page";
+import Quize from "../components/Quize/Quize";
```
Comment (orimaro123) [inline src/client/src/routes/QuizePage.jsx:2]: Let's use just Quiz for convention, why the 'e' at the end?

**Review by orimaro123 — COMMENTED** [review]
Cool that this is in the air!
Let's use Quiz instead of Quize for convention before the project becomes huge
And only where we need the app name it will be Quizy

File: src/client/src/index.css [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+body{
+  height: 100vh;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
```
Comment (asshishkova) [inline src/client/src/index.css:7]: Wow :) Maybe we do not need all of them? Or just in case?

File: src/client/src/index.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```
Comment (asshishkova) [inline src/client/src/index.js:1]: I completely agree with double quotes!

File: src/client/src/routes/BrainMatesPage.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import BrainMates from "../components/BrainMates/BrainMates";
+import Page from "../components/Page/Page";
+
+export default function BrainMatesPage() {
+  return (
+    <Page>
+      <BrainMates />
```
Comment (asshishkova) [inline src/client/src/routes/BrainMatesPage.jsx:7]: Thank you for saving the naming :)

**Pull request merged**